components/knative-eventing/base/kustomization.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/knative/eventing/releases/download/knative-v1.15.3/eventing-core.yaml?timeout=90s
```

Comment:
For the RH instances here, do we want to use the RH product version of knative eventing instead of the upstream build? I would assume we want to unless we've had deliberate conversations that concluded to the contrary.

Reply:
KubeArchive needs Knative Eventing v1.15 and the latest OpenShift Serverless version is not there yet. It may be when we reach internal instances, so we may decide to use the OpenShift Serverless variant. For now we should stick to the upstream project.

Reply:
Ack. @hugares any opinion on this from your team's pov?

Reply:
for  Tekton, we use Openshift Pipelines so following that, we should use OpenShift Serverless  but if serveless do not have the knative version we need, we have no other choice to go with upstream